backend/src/test/java/com/epam/todo/controller/TodoControllerTest.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.todo.controller;
+
+import com.epam.todo.service.TodoFacade;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+class TodoControllerTest {
+
+    @InjectMocks
+    private TodoController underTest;
+
+    @Mock
+    private TodoFacade todoFacade;
+
+    @Test
+    void testTodoController_ConstructorShouldThrowNullPointerException_WhenCalledWithNullPointer() {
+
+        assertThrows(NullPointerException.class,
+                () -> new TodoController(null));
+    }
+
+    @Test
+    void getToDos() {
```

Comment:
I think you are about to create tests.
If you didn't create any please remove this empty method.

Reply:
Done.